core/controllers/base_test.py

```diff
@@ -329,7 +329,7 @@ def _extract_keys_from_json_file(self, filename):
         )).keys())
 
     def _extract_keys_from_html_file(self, filename):
-        regex_pattern = r'(I18N_[A-Z/_\d]*)'
+        regex_pattern = r'(I18N_[A-Z/_\d]*)(?=.*_I18N_IDS$).*$'
```

Comment:
Maybe worth adding a comment to explain what this is trying to do, too, so that future devs don't break it.